awx_collection/plugins/modules/inventory.py

```
@@ -167,6 +173,7 @@ def main():
     kind = module.params.get('kind')
     host_filter = module.params.get('host_filter')
     prevent_instance_group_fallback = module.params.get('prevent_instance_group_fallback')
+    opa_query_path = module.params.get('opa_query_path')
```

Ruff (0.15.2)

[error] Local variable `opa_query_path` is assigned to but never used

Remove assignment to unused variable `opa_query_path`

(F841)